api/storage_service.cc

```diff
@@ -89,6 +89,23 @@ static auto wrap_ks_cf(http_context &ctx, ks_cf_func f) {
     };
 }
 
+future<> set_tables_autocompaction(http_context& ctx, const sstring &keyspace, std::vector<sstring> tables, bool enabled) {
```

Comment:
@amnonh why do we need two ways two control auto compaction - one in the storage service and another at the column family level ?

Which ones does scylla-jmx use ? I am okay keeping both if there is a compat reason but I'd prefer to have just one

Reply:
`scylla-jmx` only uses 1 API for the column_family
```
    @Override
    public boolean isAutoCompactionDisabled() {
        log(" isAutoCompactionDisabled()");
        return client.getBooleanValue("column_family/autocompaction/" + getCFName());
    }
```
but I thought it would be cool to have it consistent with the storage service. The only difference in the naming is the underscore:

`/column_family/autocompaction/` vs `/storage_service/auto_compaction` which I would happy to reduce to 1 variant, however, it would make jmx incompat

Reply:
I would prefer not to expose two APIs for that.

If scylla-jmx is using only column family, let's please add just that one.
It looks simpler than storage service anyway